app/frontend/packs/users/registration.js

```diff
@@ -0,0 +1,9 @@
+let oneTimeError = (classNameOfElements, elementsEvent) => {
+  document.querySelectorAll(`.${classNameOfElements}`).forEach (element => {
+    element.addEventListener(elementsEvent, (event) => {
+      element.parentNode.lastChild.remove();
+    }, { once: true });
+  });
+};
```

Comment:
move to a separate module, e.g. `shared/errors.js`
```suggestion
import { oneTimeError, addShowingErrorsToField } from '../shared/errors';
```

Reply:
resolved